### PR TITLE
Add config server URI

### LINKED ISSUE #31: Add Config Server URI

### PR DESCRIPTION
**Pull request by chriskieser**
fixes #31

### CODE REVIEW

File: packages/applications/subatomic-spring-boot-2x-template.yml [hunk 1]
```diff
@@ -53,8 +53,10 @@ objects:
         spec:
           containers:
             - name: ${APP_NAME}
-              image: ' ' 
+              image: ' '
               env:
+                - name: SPRING_CLOUD_CONFIG_URI
+                  value: "http://subatomic-config-server.${DEVOPS_NAMESPACE}.svc.cluster.local"
```
Comment (kbristow) [inline packages/applications/subatomic-spring-boot-2x-template.yml:59]: Do these things work when there is no config server?

Comment (chriskieser) [reply packages/applications/subatomic-spring-boot-2x-template.yml:59]: yup

Comment (chriskieser) [reply packages/applications/subatomic-spring-boot-2x-template.yml:59]: I mean nope

Comment (kbristow) [reply packages/applications/subatomic-spring-boot-2x-template.yml:59]: Probably need to think of how we deal with that. Will approve for now.

**Pull request merged**